packages/fxa-auth-server/lib/senders/emails/templates/passwordChangeRequired/en.ftl

```diff
@@ -1,8 +1,11 @@
 passwordChangeRequired-subject = Suspicious activity detected
 passwordChangeRequired-title = Password Change Required
 passwordChangeRequired-suspicious-activity = We detected suspicious behavior on your { -product-firefox-account }. To prevent unauthorized access to your { -product-firefox-account }, we’ve disconnected all devices on your account and are requiring you to change your password as a precaution.
+passwordChangeRequired-suspicious-activity-2 = We detected suspicious behavior on your { -product-mozilla-account }. To prevent unauthorized access to your { -product-mozilla-account }, we’ve disconnected all devices on your account and are requiring you to change your password as a precaution.
 passwordChangeRequired-sign-in = Sign back into any device or service where you use your { -product-firefox-account } and follow the steps that will be presented to you.
+passwordChangeRequired-sign-in-2 = Sign back into any device or service where you use your { -product-mozilla-account } and follow the steps that will be presented to you.
 passwordChangeRequired-different-password = <b>Important:</b> Pick a different password than what you were previously using and make sure that it is different from your email account.
 passwordChangeRequired-signoff = Best,
 passwordChangeRequired-signoff-name = The { -product-firefox-accounts } team
+passwordChangeRequired-signoff-name-2 = The { -product-mozilla-accounts } team
```

Comment:
We might want to check with the content team if we're okay using `The Mozilla accounts team` or if we should avoid using `Mozilla accounts` here.

Reply:
@bcolsson I got signoff from Vesta on removing the signoff block entirely from this email template 👍